help_to_heat/frontdoor/views.py

```diff
@@ -806,7 +813,107 @@ def get_context(self, request, session_id, *args, **kwargs):
         eligible_schemes = eligibility.calculate_eligibility(session_data)
         _ = interface.api.session.save_answer(session_id, "schemes", {"schemes": eligible_schemes})
         eligible_schemes = tuple(schemas.schemes_map[scheme] for scheme in eligible_schemes if not scheme == "ECO4")
-        return {"eligible_schemes": eligible_schemes}
+        supplier_name = SupplierConverter(session_id).get_supplier_on_general_pages()
+
+        is_in_park_home = session_data.get("park_home", "No") == "Yes"
+        is_social_housing = session_data.get("own_property") == "No, I am a social housing tenant"
+
+        # Edge case discussed as part of PC-1118
```

Comment:
Think this wants to be a bit more descriptive: specifically what are we doing and why. Did you get around to making a ticket for the longer-term issue? Would be good to leave a TODO against that as discussed.

Reply:
https://beisdigital.atlassian.net/browse/PC-1191 I have made it now, it's PC-1191. Please let me know fi there's any changes needed to the ticket.